tests/network/localnet/test_localnet.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+from libs.net.traffic_generator import is_tcp_connection
+from utilities.virt import migrate_vm_and_verify
+
+
+@pytest.mark.ipv4
+@pytest.mark.polarion("CNV-11775")
+def test_connectivity_over_migration_between_localnet_vms(nncp_localnet, localnet_server, localnet_client):
```

Comment:
I think it might be confusing that the nncp is called here but not used and I'm afraid that in the future it can be mistakenly removed from here.
I think it should be called to in the NAD and not in the test. I'll do that in a follow up PR if agreed.